NebulaWorld/Logistics/StationUIManager.cs

```diff
@@ -153,7 +153,8 @@ private static void UpdateSettingsUIBackground(StationUI packet, PlanetData pDat
             // only host should update everything
             if (!LocalPlayer.IsMasterClient)
             {
-                return;
+                // this is changed in pr #269 anyways to reflect this logic (update for everyone, not just host. after all why should'nt we)
+                //return;
             }
```

Comment:
Should remove entire if, since its not used anymore

Reply:
Yes, wanted to do the merge conflict resolve later